bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -149,7 +150,17 @@ export default class App extends Component {
     return false;
   }
 
+  renderSoundUnreadeMessages() {
```

Comment:
Theres nothing being render in this function therefore makes no sense to render it.

Probably you should check in the `componentDidUpdate` if you `hadUnreadMessages` and then perform the play action.